### DISCUSSION TITLE
Separate setting for enemy and ally speed

### DISCUSSION THREAD

**Discussion by ghost** [ihhub/fheroes2#7577] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

It's often tiresome to watch your CPU allies make a turn, even more so on large maps where they would have lots of heroes. But I can't really set their speed to highest, as it also applies to enemies whose moves I need to see.

### Describe the possible solution

Make a separate setting for AI enemy and ally speed.

### Additional info

I want to take a try at implementing this.

**Comment by ihhub** [comment]
This idea requires UI mockup first as well as proper description of the new and old options according to [these requirements](https://github.com/ihhub/fheroes2/wiki/Adding-a-new-option-into-the-game).

Therefore, I am moving this issue into the discussion section.
